### ISSUE TITLE
Parameterize dependency to user namespace

### ISSUE THREAD

**Issue by viesti** [james-henderson/simple-brepl#13]
tl;dr

Would it be possible to add parameter to specify into which namespace we do interning here:

https://github.com/james-henderson/simple-brepl/blob/3f83bbad4a5d291eb6370241a68f08873948a1a6/simple-brepl-core/src/simple_brepl/service.clj#L14

```
(defn load-brepl! [ip port]
  (reset! !brepl-config {:ip ip :port port})
  (intern 'user 'simple-brepl.....
```

Lately I had problems with having a user.clj that depends on my application which has cljx code. It requires some juggling to get cljx to run before user.clj can be loaded by Clojure runtime itself (ok, this may go away if the get cljx like feature expressions into Clojure core).

So I tried not having a `user.clj` at all (just renaming it and using `:init-ns` to load it), but simple-brepl has a dependency to `user` namespace.

Sorry for the monolog, you can close this issue at your will, just wanted to say this idea out loud :)

**Comment by jarohen** [comment]
Hi Kimmo - thanks for the report - absolutely agree that the interning's a problem. Will pick this up when I get a moment!

James
